templates/Collector/run.html.twig

```diff
@@ -0,0 +1,23 @@
+{% if result is not empty %}
+    <table>
+        <thead>
+        <tr>
+            {% for key, value in result[0] %}
+                <th>{{ key }}</th>
+            {% endfor %}
+        </tr>
+
+        </thead>
+        <tbody>
+        {% for row in result %}
+            <tr>
+                {% for key, value in row %}
+                    <td>{{ value }}</td>
+                {% endfor %}
+            </tr>
+        {% endfor %}
+        </tbody>
+    </table>
+{% else %}
+    Result is empty
+{% endif %}
```

Comment:
this was also copied (explain.html.twig), please modularize through block/embed it if you believe some stuff have to be different